base/dict.jl

```diff
@@ -189,6 +189,16 @@ function grow_to!{K,V}(dest::Associative{K,V}, itr, st)
     return dest
 end
 
+
+"""
+    similar(d::Dict, [element_type=Pair{K,V}])
```

Comment:
Also, no brackets around the second argument, the `=` is enough to know there's a default value.